packages/manager/modules/pci/src/projects/project/private-networks/add/add.html

```diff
@@ -102,14 +102,16 @@
                                     type="text"
                                     name="{{:: region + '-range-start' }}"
                                     data-ng-model="subnet.start"
-                                    disabled>
+                                    disabled
+                                    style="min-width: 0">
```

Comment:
Avoiding inline style would be better.

Putting this rule (with a comment) to one of these file would be great:

- manager/packages/manager/modules/pci/src/projects/project/project.less
- manager/packages/manager/modules/pci/src/index.scss

Is that make sense to you?

Reply:
This is something specific to the private network creation so it doesn't make sense to have it there